OPERATOR.rst

```diff
@@ -666,6 +665,134 @@ one branch (see ``deployments/*.browser/environment.py``) and it will always use
 the tarball from the most recent pipeline on that branch.
 
 
+Running a ZAP vulnerability scan
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ZAP Setup
+"""""""""
+
+- Download ZAP from https://www.zaproxy.org/
+
+- Install & open ZAP
+
+- From the popup, select ``No, I do not want to persis this session at this
+  moment in time`` and click ``Start``
+
+- From the menu, select ``Edit`` -> ``ZAP Mode`` -> ``Standard Mode`` (This
+  should be the default option)
+
+- From the menu, select ``Tools`` -> ``Options``:
+
+  - -> ``Network`` -> ``Rate Limit``, add and enable a 3 request per second rule
+    for the match string ``anvilprod.org``, and another rule for the match
+    string ``humancellatlas.org``. This is required to prevent the scans from
+    exceeding the WAF rate limits for the Azul APIs, and being temporarily
+    blocked by the wAF.
+
+  - -> ``Check for Updates``:
+
+    - Check for updates on startup: Checked
+
+    - Check for updates to the add-ons you have installed: Checked
+
+Running an authenticated scan
+"""""""""""""""""""""""""""""
+
+The process for running an authenticated scan is to first obtain an Azul
+authentication token, and launch the ZAP application with the token set as
+an environment variable. See
```

Comment:
This is reStructuredText

https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages